include/boost/geometry/iterators/point_reverse_iterator.hpp

```diff
@@ -29,16 +29,6 @@ class point_reverse_iterator
 private:
     typedef std::reverse_iterator<point_iterator<Geometry> > base;
```

Comment:
Both `g++` and `clang++` confuse the `base` type with the `base()` method called in the constructor.
Please rename `base` to `base_type`.